Dockerfile

```diff
@@ -18,8 +18,10 @@ RUN rm ./target/release/deps/vaultwarden_ldap*
 COPY src ./src
 RUN cargo build --release
 
-FROM rust:$RUN_TAG
+FROM ubuntu:focal
```

Comment:
Use latest LTS tag: `ubuntu:22.04`

Reply:
Ah. I see. So the current Docker Rust image I'm using only works with the `focal` base version of `libssl-dev`. If you can clean up the other change and still target this image, I'm ok with that. I have a separate goal of upgrading the Rust version, but I can't get the Alpine image to work. I may end up abandoning it.

Reply:
Yea I hit similar issues which is what led me to have to use focal